File: js/PROBLEMS/1149.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split(' ')

lines = lines.map(x => parseInt(x))
a = lines[0]
for (i of lines){

    if (i > 0){
        n=i
    }
}

r = 0;
for (i=1;i<=n;i++){
    r+=a;
    a++;
}
console.log(r+'')

File: python/PROBLEMS/1149.py
numbers = list(map(lambda x: int(x), input().split(' ')))
a = numbers[0]
n = next((x for x in numbers[1:] if x > 0), None)

if n<=0:
    while True:
        n = int(input())
        if n>0:
            break

r = 0

for i in range(1, n+1):
    r+=a
    a+=1

print(r)